src/python/nimbusml/internal/entrypoints/models_onnxconverter.py

```diff
@@ -1,116 +0,0 @@
-# - Generated by tools/entrypoint_compiler.py: do not edit by hand
-"""
-Models.OnnxConverter
```

Comment:
is possible to keep this one in?

Reply:
In order to keep this one the `Microsoft.ML.OnnxConverter` package would have to be added as a new package reference in the *.csproj files. This is done in the pending onnx converter pull request ([https://github.com/microsoft/NimbusML/pull/227](https://github.com/microsoft/NimbusML/pull/227)). Do you want to add a dependency for this new library now or wait till the onnx converter code is added?

---
In reply to: [334715219](https://github.com/microsoft/NimbusML/pull/329#discussion_r334715219) [](ancestors = 334715219)